console.log("firest line");
console.log("second");
try{
// let a=10;
console.log("value of a is:",a);
}
catch(err)
{
console.log("error has occured");
  
}
console.log("third line");

function test()
{
    console.log("test is calle");
}
test();
